conf/modules.config

```diff
@@ -774,14 +773,15 @@ process {
 
     withName: TAXPASTA_STANDARDISE {
         tag = { "${meta.tool}|${meta.id}" }
+        ext.prefix = { "${meta.tool}_${meta.id}" }
         ext.args =  {
-            [
-                "-p ${meta.tool} -o ${meta.tool}_${meta.id}.${params.standardisation_taxpasta_format}",
-                params.taxpasta_add_name ?  "--add-name" : "",
-                params.taxpasta_add_rank ? "--add-rank" : "",
-                params.taxpasta_add_lineage ? "--add-lineage" : "",
-                params.taxpasta_add_idlineage ? "--add-id-lineage" : ""
-            ].join(' ').trim()
+                [
```

Comment:
I am somewhat confused here, don't we need to give the profiler argument?

Reply:
It has moved to the module invocation itself, supplied via a multimap!